dds/DCPS/RTPS/Spdp.cpp

```diff
@@ -3100,6 +3100,8 @@ Spdp::SpdpTransport::handle_input(ACE_HANDLE h)
       message.hdr = header;
     }
 
+    DCPS::GuidPrefix_t destinationGuidPrefix = {0};
```

Comment:
```suggestion
    DCPS::GuidPrefix_t destinationGuidPrefix = DCPS::GUIDPREFIX_UNKNOWN;
```

Reply:
Also, it appears that this variable is not set.

Reply:
> (diff with GUID_PREFIX_UNKNOWN)

Unfortunately, arrays can't be initialized that way.